packages/babel-preset-env/test/fixtures/corejs3/entry-entries-modules-chrome-71/output.mjs

```diff
@@ -1,2 +1,2 @@
-import "core-js/modules/es.object.from-entries.js";
-import "core-js/modules/esnext.string.replace-all.js";
+import 'core-js/modules/es.object.from-entries';
+import 'core-js/modules/esnext.string.replace-all';
```

Comment:
Can you add test cases when input file contains core-js imports ending with `.js`?

Reply:
Makes sense 👍  is this single scenario enough or you had more cases in mind?
https://github.com/babel/babel/pull/12554/commits/28f6547f8d593ccb4001650e154fcbaf4c0fc386